eng/common/scripts/git-branch-push.ps1

```diff
@@ -60,6 +65,16 @@ if ($LASTEXITCODE -ne 0)
     exit $LASTEXITCODE
 }
 
+if ($SkipCommit -and $AmendCommit) {
```

Comment:
I would not make this a separate step I would simply add `--amend` to the other commit if AmendCommit is set to true.